Cargo.toml

```diff
@@ -51,6 +51,8 @@ codespan-reporting = "0.11"
 phf = "0.10.1"
 rust-lapper = "1.0"
 bitflags = "1.3"
+anchor-syn = { version = "0.25", features = ["idl"] }
```

Comment:
We could do. I think we should wait until there is a real demand for this, not just hypothetical.

Reply:
Well it will help to reduce compile time. But we can introduce them later and also think a bit more about the architecture of the flags we want to have.